server/src/test/java/org/elasticsearch/cluster/metadata/DataLifecycleTests.java

```diff
@@ -39,19 +40,16 @@ protected Writeable.Reader<DataLifecycle> instanceReader() {
 
     @Override
     protected DataLifecycle createTestInstance() {
-        if (randomBoolean()) {
-            return new DataLifecycle();
-        } else {
-            return new DataLifecycle(randomMillisUpToYear9999());
-        }
+        return switch (randomInt(2)) {
```

Comment:
I don't think that any of the objects returned from this method have `nullified` set to `true`, do they? So we're missing some coverage.

Reply:
Oh you actually have a comment about this later in the code. It might be worth mentioning it here, too, so that someone in the future doesn't come along and try adding a nullified instance here.